spec/models/mixins/ownership_mixin_spec.rb

```diff
@@ -0,0 +1,76 @@
+describe OwnershipMixin do
+  let(:user) { FactoryGirl.create(:user_with_email_and_group) }
+  let(:other_user) { FactoryGirl.create(:user) }
+  let(:group) { user.current_group }
+  let(:vm) { FactoryGirl.create(:vm) }
+  let(:vm_with_user) { FactoryGirl.create(:vm, :evm_owner => user) }
+  let(:vm_with_group) { FactoryGirl.create(:vm, :miq_group => group) }
+
+  describe ".evm_owner_email" do
+    it "shows blank user" do
```

Comment:
While technically correct, it what the user is seeing is `it "returns nil when there is no user associated"`.